blockchain/Dockerfile

```diff
@@ -1,29 +1,31 @@
-FROM node:10.5-slim
+FROM node:18.4.0-alpine
+
+RUN deluser --remove-home node
 
 # Default P2P Port:
 EXPOSE 7447
 # Default RPC Port:
 EXPOSE 8000
 
+# Download Multichain
+RUN apk add --no-cache curl
 RUN curl -ko- https://www.multichain.com/download/multichain-2.1.2.tar.gz | tar xzv -C /usr/local/bin --strip 1 --exclude='multichain-2.1.2/multichaind-cold'
 
-# Since we want to store private keys away from the network
-#RUN rm  /usr/local/bin/multichaind-cold
+RUN mkdir -p /root
+RUN mkdir -p /home/node
 
 WORKDIR /home/node
-
-COPY package*.json ./
-
+COPY package.json ./
+COPY package-lock.json ./
 RUN npm ci
+COPY ./src/ ./src/
+RUN chmod +x ./src/multichain-feed/multichain-feed
 
-COPY src/ src/
-RUN chmod +x src/multichain-feed/multichain-feed
 
 ARG BUILDTIMESTAMP=''
 ARG CI_COMMIT_SHA=''
-
 ENV BUILDTIMESTAMP ${BUILDTIMESTAMP}
 ENV CI_COMMIT_SHA ${CI_COMMIT_SHA}
 
-# Run Chain
+# Start as root user: Multichain needs root access to the file system
```

Comment:
You sure? Can not find anything about this in their documentation, maybe `chown`/`chmod` the multichain dir?